src/main/java/spark/Routable.java

```diff
@@ -0,0 +1,772 @@
+/*
+ * Copyright 2015 - Per Wendel
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package spark;
+
+import spark.route.HttpMethod;
+import spark.utils.SparkUtils;
+
+/**
+ * Routable abstract class. Lets extending classes inherit default routable functionality.
```

Comment:
I did start with that but I fail(ed) to see how that would make things better in this case.